.travis.yml

```diff
@@ -10,29 +10,26 @@ env:
   global:
   - TRAVIS_RUST_VERSION=nightly-2019-07-01
   - RUST_BACKTRACE=1
-  - TWINE_USERNAME=pattonw
+  - PYPI_USERNAME=pattonw
 
 test_template: &test_template
   stage: test
   before_install:
     source ./ci/travis/setup_rust.sh
```

Comment:
since the test.sh and build.sh files are now replaced with make commands, maybe we should add a make command for installing the rust toolchain as well and get rid of all the extra bash scripts.

Reply:
I'm a little hesitant to allow make recipes to change things about the user's system outside of the project scope; I like it being a separate script where it's clear that "this is for travis, not for you". That's a personal thing though. We could have a make recipe that checks we've got the right compiler etc, so that it fails faster and more informatively.